### DISCUSSION TITLE
How to update Syncthing Tray for Windows  ?

### DISCUSSION THREAD

**Discussion by Kermit68** [Martchus/syncthingtray#238] [Q&A] [answered]
I  swear I read all the readme and also I googled looking for this info but I could not find the proper procedure to update my current Syncthing Tray for windows. I tried to replace the previous version of the .exe file with the new one but it did not worked, I guess because the in the name there  is the version of the release so the new file is not recognised as it should be. 
How I'm supposed to proceed whithout screwing my existing Syncthing installation? (I eneded up with two separate and different istances of Syncthing ....)
Thanks

**Comment by ProactiveServices** [comment]
Without details on why it didn't work, and how it's currently being run, it's difficult to help you.

**Reply by Kermit68** [reply]
You're right, I  hoped  I missed some  "stupid"  step  so  no details  where needed  (sorry  for  the typos  by  the  way,  my  keybopard  is   dieing  ...)

**Comment by Martchus** [comment, marked as answer]
You definitely won't need to reinstall anything (Syncthing Tray itself doesn't even come with an installer). The executable can be renamed to whatever you like. When remaining it you'll have to revisit the autostart settings but the UI will guide you.

You might not have ended up with two instances of Syncthing but just two tray icons. Just close the ones you don't need.

**Reply by Kermit68** [reply]
Thanks, so I can   rename  the  .exe,  I  was   afraid   I could  mess  some  reference   to  the  file.  I'll  try that  and let you know, it may be of some help in the future for others.

**Reply by Kermit68** [reply]
Thanks again. I renamed the  .exe without making reference to the version and updated the registry key for startup.
